It's not worth introducing a new dependency for a mere sphinx theme...
diff --git a/doc/conf.py b/doc/conf.py
index ada9ec49..2bd66593 100644
--- a/doc/conf.py
+++ b/doc/conf.py
@@ -99,7 +99,7 @@ pygments_style = 'sphinx'
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'python_docs_theme'
+html_theme = 'nature'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
